src/Symfony/Component/Validator/CHANGELOG.md

```diff
@@ -11,6 +11,15 @@ CHANGELOG
  * added the `compared_value_path` parameter in violations when using any 
    comparison constraint with the `propertyPath` option.
  * added support for checking an array of types in `TypeValidator`
+ * deprecated `All` in favor of `Each` constraint
+ * deprecated `AllValidator` in favor of `EachValidator` validator
+ * added `Each` constraint
+ * added `EachValidator`
+ * added `None` constraint
+ * added `NoneValidator`
```

Comment:
I suggest merging this line with the previous one. Adding the new constraint and its validator are a single feature addition (same for other constraints).
Older versions don't mention the validator at all btw, only the new constraint (as that's the relevant user-facing thing)